pom.xml

```diff
@@ -35,11 +35,24 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.mysql</groupId>
+			<artifactId>mysql-connector-j</artifactId>
+			<version>8.4.0</version>
+		</dependency>
+
+
 		<dependency>
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
 			<optional>true</optional>
```

Comment:
Doczytaj co to robi i czy czasem nie byłoby bezpieczniej tu usunąć (<optional>)

Reply:
"W zapisie Maven, tag <optional>true</optional> w sekcji <dependency> ma na celu określenie, że dana zależność jest opcjonalna. Oznacza to, że jeśli projekt A zależy od projektu B, który ma oznaczoną zależność C jako opcjonalną, to projekt A nie będzie automatycznie wymagał zależności C."
Nie wiem wiec czy lepiej tego po prostu sobie nie zostawić